staging/src/k8s.io/apiserver/pkg/cel/openapi/resolver/refs.go

```diff
@@ -60,6 +60,36 @@ func populateRefs(schemaOf func(ref string) (*spec.Schema, bool), visited sets.S
 			return nil, fmt.Errorf("internal error: cannot resolve Ref %q: %w", ref, ErrSchemaNotFound)
 		}
 		result = *resolved
+
+		// Overwrite structural information like description/default into
```

Comment:
I wonder if it's best to move all this upstream (not only the addition in this PR, but this entire refs resolver). Other clients could probably benefit from the post processing done here, and with all the kube-openapi changes made to better support CEL & validation during openapi generation, mirroring these changes in kube-openapi should make logical sense.